dydx/dydxPresenters/dydxPresenters/_v4/GlobalWorkers/Workers/dydxTransferSubaccountWorker.swift

```diff
@@ -40,8 +40,8 @@ final class dydxTransferSubaccountWorker: BaseWorker {
                     subaccountNumber = 0
                 }
                 let depositAmount = (balance ?? 0) - dydxTransferSubaccountWorker.balanceRetainAmount
-                let amountString = dydxFormatter.shared.raw(number: NSNumber(value: depositAmount),
-                                                            digits: dydxTokenConstants.usdcTokenDecimal)
+                let amountString = dydxFormatter.shared.decimalRaw(number: NSNumber(value: depositAmount),
+                                                                   digits: dydxTokenConstants.usdcTokenDecimal)
```

Comment:
does formatting the amountString here serve a purpose? I would think the chain query would chop off any precision if necessary. If we can keep it as the `depositAmount` as a `Double`, can we just pass the double directly into `CosmoJavascript.shared.depositToSubaccount` in the next function? e.g.

```
    private func depositToSubaccount(amount: **Double**, subaccount: Int, walletState: dydxWalletState) {
        CosmoJavascript.shared.depositToSubaccount(subaccount: subaccount, **amount: amount**) { result in // note bolded would have to change in the declaration
```